packages/patrol/example/android/app/build.gradle

```diff
@@ -32,6 +32,13 @@ android {
     kotlinOptions {
         jvmTarget = "1.8"
     }
+    
+    //TODO verify
+    packagingOptions {
+        pickFirst "META-INF/INDEX.LIST"
+        pickFirst "META-INF/io.netty.versions.properties"
+        pickFirst "META-INF/DEPENDENCIES"
+    }
```

Comment:
I spent some time to find another solution. This problem is similar to this one (link) and others related to "META/sth".
https://stackoverflow.com/questions/44342455/more-than-one-file-was-found-with-os-independent-path-meta-inf-license
There are different answers in this topic. I think this is a configuration issue with example app. It would be crazy if khttp dependencies in the same version caused metadata conflicts. We can delete the android project and configure it again. 
We can remove plugins or even change the server implementation, but I think it would be easier to try in other projects whether this problem occurs. If you want I can ask testers to use patrol with git commit and check this without merging and prerelease version.